tests/containers/podman_pods.pm

```diff
@@ -116,28 +115,18 @@ sub run {
         validate_script_output('podman pod ps', sub { !m/testing-pod/ });
         validate_script_output('podman ps', sub { !m/testing-pod-container/ });
 
-        # Staging does not have access to repositories, only to DVD
-        # curl -sfL https://get.k3s.io is not supported on ppc poo#128456
-        if (check_min_runtime_version('4.4.0') && !is_staging && !is_ppc64le) {
-            install_k3s();
-            record_info('Test', 'kube apply');
-            assert_script_run('podman kube apply --kubeconfig ~/.kube/config -f pod.yaml', timeout => 180);
-            assert_script_run('kubectl wait --timeout=600s --for=condition=Ready pod/testing-pod', timeout => 610);
-            validate_script_output('kubectl exec testing-pod -- cat /etc/os-release', sub { m/SUSE Linux Enterprise Server/ }, timeout => 300);
-        }
     }
 
     if (check_min_runtime_version('4.4.4')) {
         check_container_nspid();
         check_container_nspid(1);
     }
+
 }
 
 sub cleanup {
     my ($self) = @_;
     $self->{podman}->cleanup_system_host();
-    # Staging does not have access to repositories, only to DVD
-    uninstall_k3s() if (check_min_runtime_version('4.4.0') && !is_staging && !is_ppc64le);
```

Comment:
Don't we also need to move this part somewhere?

Reply:
@ricardobranco777 Since in `podman_pods` we do not use k3s it is not needed there and `run_container_in_k3s` the `uninstall_k3s `in happening anyway.

Reply:
> @ricardobranco777 Since in `podman_pods` we do not use k3s it is not needed there and `run_container_in_k3s` the `uninstall_k3s `in happening anyway.

Then we can remove this line in the old file:
`use containers::k8s qw(install_k3s uninstall_k3s);`

Reply:
Let's still do this, then we can merge.